docker/server/config/config-mysql.properties

```diff
@@ -12,12 +12,20 @@ spring.datasource.password=conductor
 spring.datasource.hikari.maximum-pool-size=10
 spring.datasource.hikari.minimum-idle=2
 
+# disabled: it causes error after second docker-compose up
 # Elastic search instance indexing is enabled.
-conductor.indexing.enabled=true
+# conductor.indexing.enabled=true
```

Comment:
This needs to be uncommented to enable Elasticsearch, which i believe is the intention.

Reply:
@imcyee this is the last change requested before the PR can be merged.